state-chain/runtime/src/chainflip/decompose_recompose.rs

```diff
@@ -25,20 +21,8 @@ fn decode_many<T: Encode + Decode>(data: &mut [Vec<u8>]) -> Vec<T> {
 }
 
 fn select_median_btc_info(data: Vec<BitcoinFeeInfo>) -> Option<BitcoinFeeInfo> {
```

Comment:
I would say now we don't even need this function, and we can inline it - the tests for select_median_btc_info are basically just testing the `select_median` function now, so would delete them

Reply:
I'd agree if the `Ord` would be defined for `BitcoinFeeInfo`.
Right now it can be defined as just a forward to `Ord::cmp(&self.sats_per_kilo_byte, &other.sats_per_kilo_byte)`.
But if some other data is added into `BitcoinFeeInfo`, such a definition of `Ord for BitcoinInfo` may become surprising.

Reply:
fair